MapboxAndroidDemo/src/main/res/layout/activity_location_basic.xml

```diff
@@ -21,7 +21,8 @@
         android:layout_height="wrap_content"
         android:layout_gravity="end|bottom"
         android:layout_margin="16dp"
-        android:src="@drawable/ic_my_location_24dp"
+        app:srcCompat="@drawable/ic_my_location_24dp"
```

Comment:
Any reason behind changing this to compat? Either `src` or `srcCompat` works on my end.